content/docs/command-reference/check-ignore.md

```diff
@@ -0,0 +1,83 @@
+# check-ignore
+
+Used for debugging `.dvcignore` / exclude files.
+
+## Synopsis
+
+```usage
+usage: usage: dvc check-ignore [-h] [-q | -v] [-d] [-n]
+                               targets [targets ...]
+
+positional arguments:
+  targets        Input files/directories to check ignore patterns.
+```
+
+## Description
+
+For each pathname given via the command-line , check whether the file is excluded 
+by `.dvcignore` output the path if it is excluded.
+
+## Options
+
+- `-h`, `--help` - prints the usage/help message, and exit.
+
+- `-q`, `--quiet` - do not write anything to standard output. Exit with 0 if no
+  problems arise, otherwise 1.
+
+- `-v`, `--verbose` - displays detailed tracing information.
+
+- `-d`, `--details` - show the exclude pattern together with each target path.
+
+- `-n`, `--non-matching` - show the target paths which don’t match any pattern.
+  Only usable when `--details` is also employed
```

Comment:
Let's put these 2 on top, before -h -q -v.

Reply:
Resolved in https://github.com/iterative/dvc.org/pull/1629#pullrequestreview-462178601